Public/Invoke-AtomicTest.ps1

```diff
@@ -203,6 +203,23 @@ function Invoke-AtomicTest {
         }
 
         if($isLoggingModuleSet) {
+            # Since there might a comma(T1559-1,2,3) Powershell takes it as array.
+            # So converting it back to string.
+            if($AtomicTechnique -is [array]) {
+                $AtomicTechnique = $AtomicTechnique -join ","
+            }
+            
+            # Splitting Atomic Technique short form into technique and test numbers.
+            $AtomicTechniqueParams =  ($AtomicTechnique -split '-')
+            $AtomicTechnique = $AtomicTechniqueParams[0]
+
+            if($AtomicTechniqueParams.Length -gt 1){
```

Comment:
When I use `Invoke-AtomicTest T1554.003` AtomicTechniqueParams would be `["T1554.003"]`. Here both the first and last element of the array would be same. So adding a length check